src/wazuh_modules/inventory_harvester/src/fimInventory/fimContext.hpp

```diff
@@ -674,6 +675,17 @@ struct FimContext final
         return m_pathSanitized;
     }
 
+    std::string_view hashPath()
+    {
+        if (m_pathHashed.empty())
+        {
+            Utils::HashData hash(Utils::HashType::Sha256);
+            hash.update(m_pathSanitized.c_str(), m_pathSanitized.size());
+            m_pathHashed = Utils::asciiToHex(hash.hash());
+        }
```

Comment:
This implementation not work, we are hash an empty string in all cases.
```suggestion
        if (m_pathHashed.empty())
        {
            auto pathRawStr = pathRaw();
            Utils::HashData hash(Utils::HashType::Sha256);
            hash.update(pathRawStr.data(), pathRawStr.size());
            m_pathHashed = Utils::asciiToHex(hash.hash());
        }
```